src/components/PeopleTableHead/PeopleTableHead.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from '../SearchLink/SearchLink';
+import { OrderByType, SortBy } from '../../types/OrderAndSortTypes';
+
+const sortByValues = ['Name', 'Sex', 'Born', 'Died'];
+
+export const PeopleTableHead: FC = () => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const changeSortAndOrder = (sortByV: SortBy, orderByV: OrderByType) => {
+    const lowercaseSort = sortByV.toLowerCase();
+
+    switch (orderByV) {
+      case '':
+        return {
+          sort: lowercaseSort || null,
+          order: 'asc',
+        };
+
+      case 'asc':
+        return {
+          sort: lowercaseSort || null,
+          order: sort === lowercaseSort ? 'desc' : 'asc',
+        };
+
+      case 'desc':
+        return {
+          sort: sort === lowercaseSort ? '' || null : lowercaseSort,
+          order: sort === lowercaseSort ? '' || null : 'asc',
+        };
+
```

Comment:
It would be very cool to put such strings into separate variables to avoid using magic data.
For example:
```
const EMPTY = '';
const ASCENDING = 'asc';
const DESCENDING = 'desc';
```
It may not seem essential, but such things greatly improve the readability of the code and prevent errors.
This applies to all code.